code/modules/clothing/glasses/glasses.dm

```diff
@@ -120,18 +122,21 @@
 	icon_state = "glasses"
 	item_state = "glasses"
 	vision_correction = 1 //corrects nearsightedness
+	cosmetic = TRUE
 
 /obj/item/clothing/glasses/regular/hipster
 	name = "Prescription Glasses"
 	desc = "Made by Uncool. Co."
 	icon_state = "hipster_glasses"
 	item_state = "hipster_glasses"
+	cosmetic = TRUE
 
 /obj/item/clothing/glasses/gglasses
 	name = "Green Glasses"

```

Comment:
These are removed in my other PR, and their sprite was never improved/one of the glasses sprites that wasn't redone.  Guess we'll have to wait and see if my PR that removes them is even merged or what not and go from there.